neqo-transport/src/connection/params.rs

```diff
@@ -121,6 +123,9 @@ impl Default for ConnectionParameters {
             disable_migration: false,
             pacing: true,
             pmtud: false,
+            // Firefox on OpenBSD is sandboxed and cannot access the routing socket that the mtu
+            // crate uses to obtain the local interface MTU. (Trying to do so crashes the process.)
```

Comment:
Maybe link to [bug 1952304](https://bugzilla.mozilla.org/show_bug.cgi?id=1952304) for more context?

Reply:
Given that the OpenBSD issue is Firefox specific, I prefer moving the `cfg!(not(target_os = "openbsd"))` to `neqo_glue`, and instead, set the default to always `true` here in `neqo-transport`.

We can then link to the Bugzilla Bug in `neqo_glue`.